src/openforms/submissions/form_logic.py

```diff
@@ -90,10 +95,11 @@ def evaluate_form_logic(
         submission.form._cached_logic_rules = rules
 
     submission_state = submission.load_execution_state()
-
+    
     updated_data = deepcopy(step.data)
     for rule in rules:
         if jsonLogic(rule.json_logic_trigger, data):
+            logevent.submission_logic_evaluated(submission, rule, trigger=True)
```

Comment:
hrm, we only log the rule if it evaluates to true, but not if it's not triggered. That doesn't seem right to me - I'd expect all the evaluated rules to be logged.